common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1030,6 +1064,14 @@ void submitCleanupTask(Runnable task) {
     mergedShuffleCleaner.execute(task);
   }
 
+  /**
+   * Check `mergedShuffleCleaner` is null or already shutdown.
```

Comment:
Nit: Looks like it is only checking whether the ExecutorService is already shutdown?

Reply:
fixed